libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -86,15 +87,20 @@ - (void)setUp {
     // Delete the content of the global library directory
     NSString *globalLibraryDirectory = [[SFDirectoryManager sharedManager] directoryForUser:nil type:NSLibraryDirectory components:nil];
     [[[NSFileManager alloc] init] removeItemAtPath:globalLibraryDirectory error:nil];
-
     // Set the oauth client ID after deleting the content of the global library directory
     // to ensure the SFUserAccountManager sharedInstance loads from an empty directory
     self.uam = [SFUserAccountManager sharedInstance];
-    self.uam.oauthClientId = @"fakeClientIdForTesting";
+    [SFAuthenticationManager sharedManager].oauthClientId = @"fakeClientIdForTesting";
```

Comment:
This is another area where overriding this value means that we'd need to persist the original value and rehydrate it on tearDown, to ensure the persisted data of the singleton.

But in this case, I'd recommend simply removing this setting altogether, and just inline-set it in `createNewUserWithIndex:`, the only place it's used in these tests.  The credentials created in here are isolated "local" objects for unit testing purposes—there's no need to tie to an app-level ClientID value to utilize them.